lib/starknet-jvm.md

```diff
@@ -393,6 +413,39 @@ fun main(args: Array<String>) {
         low = response[0],
         high = response[1],
     )
+
+    // Create multiple calls
```

Comment:
I don't have a strong opinion on this, but I think:
1. This should be a separate example
2. We should send calls to separate addresses, or better, to separate endpoints

Reply:
I moved it to separate example and changed the addresses. Example of separate methods usage can't be covered, as we currently don't support multiple methods yet.